install/scripts/cli.py

```diff
@@ -0,0 +1,177 @@
+def add(import_, counselors):
+    """Add new counselors in NAME=PUBLIC_KEY format to server database"""
+    counselors += tuple(f"{file.name}={file.read()}" for file in import_)
+    counselors = dict(counselor.strip(",").split("=") for counselor in counselors)
+    [
+        click.echo(f"counselor {name} added with public key: {pub_key}")
+        for name, pub_key in counselors.items()
+    ]
+
+
+@admin.command()
+@click.confirmation_option(prompt="are you sure you want to remove these users?")
+@click.option("--pubkey", "-k", multiple=True, help="delete counselor by public key")
+@click.option(
+    "--file",
+    "-f",
+    type=click.File(),
+    multiple=True,
+    help="delete counselor by file used to import",
+)
+@click.option("--all", "all_", is_flag=True, help="delete entire counselor database")
+@click.argument("names", nargs=-1)
+def remove(pubkey, file, all_, names):
+    """remove counselor NAMES from server database"""
+    if all_:
+        click.confirm(
+            "this will remove all counselors from the database\n"
+            "Are you really sure this is what you want?",
+            abort=True,
+        )
+        click.echo("all counselors deleted")
+
+    [click.echo(f"found counselor with key: {key}") for key in pubkey]
+    [click.echo(f"found counselor from file {f.name}") for f in file]
+    [click.echo(f"counselor removed: {name.strip(',')}") for name in names]
+
+
+def load_config(ctx, param, value):
+    if not value:
+        return
+    click.echo(f"settings loaded from {value.name}")
+    ctx.exit()
+
+
+def save_config(ctx, param, value):
+    if not value:
+        return
+    click.echo(f"settings stored to {value.name}")
+    ctx.exit()
+
+
+def default_config(ctx, param, value):
+    if not value:
+        return
+    click.confirm(
+        "This will erase all active settings!\n"
+        "It is recommended you first backup your current settings with --export\n"
+        "Set all configuations to default?",
+        abort=True,
+    )
+    click.echo("All settings changed to defaults")
+    ctx.exit()
+
+
+@admin.command()
+@click.option(
+    "--import",
+    "-i",
+    is_eager=True,
+    type=click.File(),
+    expose_value=False,
+    help="populate config from a file",
+    callback=load_config,
+)
+@click.option(
+    "--export",
+    "-e",
+    is_eager=True,
+    expose_value=False,
+    type=click.File("w"),
+    help="write existing configuration to a file",
+    callback=save_config,
+)
+@click.option(
+    "--default",
+    "-d",
+    type=str,
+    multiple=True,
+    help="set the named setting to its default value",
+)
+@click.option(
+    "--all_defaults",
+    is_flag=True,
+    is_eager=True,
+    expose_value=False,
+    help="set all config values to application defaults",
+    callback=default_config,
+)
+@click.argument("settings", nargs=-1)
+def config(default, settings):
+    """change settings in KEY=VALUE format or import/export config file"""
```

Comment:
There's a bunch of capitalization problems in this PR but I guess it doesn't matter :man_shrugging:

Reply:
I knew the grammar was inconsistant but I stopped caring because I knew I'd be back here in a week to actually implement the actions